addons/payment/tests/test_acquirer.py

```diff
@@ -0,0 +1,80 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+
+from odoo.tests import tagged
+
+from odoo.addons.payment.tests.common import PaymentCommon
+
+
+@tagged('-at_install', 'post_install')
+class TestAcquirerPayment(PaymentCommon):
+
+    def _get_acquirers(self, amount=None):
+        return self.acquirer._get_compatible_acquirers(
+            company_id=self.company.id,
+            partner_id=self.partner.id,
+            currency_id=self.currency.id,
+            force_tokenization=False,
+            amount=amount
+        )
+
+    def test_compatible_acquirers_returns_on_no_amount(self):
```

Comment:
Maybe you could add a test that no acquirer is returned when the amount is zero?

Reply:
done :)